pact/deleteAssessmentAssignee.pact.test.ts

```diff
@@ -0,0 +1,71 @@
+// @ts-nocheck
+import { pactWith } from 'jest-pact'
+import { Matchers } from '@pact-foundation/pact'
+import { deleteAssessingUser } from '../server/clients/manageRecallsApi/manageRecallsApiClient'
+import * as configModule from '../server/config'
+import getRecallResponseJson from '../fake-manage-recalls-api/stubs/__files/get-recall.json'
+import { pactJsonResponse, pactDeleteRequest } from './pactTestUtils'
+
+pactWith({ consumer: 'manage-recalls-ui', provider: 'manage-recalls-api' }, provider => {
+  const accessToken = 'accessToken-1'
+  const userId = '00000000-0000-0000-0000-000000000000'
+  const recallId = '123-456'
+
+  beforeEach(() => {
+    jest.spyOn(configModule, 'manageRecallsApiConfig').mockReturnValue({ url: provider.mockService.baseUrl })
+  })
+
+  describe('unassign user from recall assessment', () => {
+    test('can assign a user', async () => {
+      await provider.addInteraction({
+        state: 'a user can be unassigned',
+        ...pactDeleteRequest('an unassign user request', `/recalls/${recallId}/assignee/${userId}`, accessToken),
+        willRespondWith: pactJsonResponse(Matchers.like(getRecallResponseJson), 200),
```

Comment:
should this return a different json? One WITHOUT an assignee?

Reply:
Done